src/components/Article.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Article({ title, date, text }) {
```

Comment:
I think you should move each parameter to a new line. For example:
export function Article({
  title,
  date,
  text
})